capstone/src/main/resources/static/map.js

```diff
@@ -5,7 +5,7 @@ async function createMap() {
     document.getElementById('map'),
     {center: princetonLatLng, zoom: 16});
 
-   const response = await fetch('get-all-events');
+   const response = await fetch('get-map-events');
```

Comment:
Oh in one of my prs I created a new mapping function for filtered events instead and left all-events as normal bc of the error it was causing on maps. So I think whatever pr gets pushed in first, we can just adjust to those mapping functions?

Reply:
So none of you guys use 'get-all-events' anymore? I have a PR that filters out past events for that end point but if you guys are not using it I'll just wait for these functions to be merged.

Reply:
I think filtering out the past events is really useful, so I'm fine with using that instead